### PR TITLE
0 ace and culture

### PR DESCRIPTION
**Pull request by rajkeppour**
- Added Imperial Culture
- TODO - create custom Ace graphics and reference the new files in interface/aces.gfx

### CODE REVIEW

File: common/graphicalculturetype.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+western_european_2d
+eastern_european_2d
+asian_2d
+southamerican_2d
+middle_eastern_2d
+african_2d
+sami_gfx
+sami_2d
+### EXECUTRIX ###
+imperial_2d
```
Comment (rajkeppour) [inline common/graphicalculturetype.txt:18]: New 2d culture

File: interface/aces.gfx [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+		name = "GFX_ITA_ace_Vittorio_Sanseverino"
+		texturefile = "gfx/aces/ITA_vittorio_sanseverino_ace.dds"
+	}
+	spriteType = {
+		name = "GFX_ITA_ace_Giuseppe_Cenni"
+		texturefile = "gfx/aces/ITA_giuseppe_cenni_ace.dds"
+	}
+
+
+	### EXECUTRIX ###
```
Comment (rajkeppour) [inline interface/aces.gfx:300]: TODO - create texture files for imperial aces and modify these lines to include them

File: interface/ideas.gfx
Comment (rajkeppour) [inline interface/ideas.gfx:1]: Not sure if we need to copy this whole file, new to HOI4 modding. Added an imperial culture that uses western_european_2d sprites as placeholders. Also not sure if we need this and don't know what happens if there is no entry for imperial culture

**Comment by rajkeppour** [comment]
This change involved additions to vanilla files graphicalculturetype.txt, aces.gfx, and ideas.gfx. Not sure what the protocol is for appending entries to vanilla files. Do we need to paste over the entire original file with the new entries? Or will a small file with only the new entries work when adding a few lines to a larger vanilla file? The latter would be better since it would be more robust to when PDX updates the vanilla files.

File: interface/imperial_culture.gfx [hunk 3]
```diff
@@ -0,0 +1,110 @@
+spriteTypes = {
+
+	### EXECUTRIX ###
+	### ACES ###
+
+	# MALE #
+	spriteType = {
+		name = "GFX_imperial_2d_ace_m_0"
+		texturefile = "gfx/aces/african_ace_m_0.dds"
```
Comment (rajkeppour) [inline interface/imperial_culture.gfx:9]: Add reference to ace sprite files here

**Review by braineac2000 — APPROVED** [review]

**Pull request merged**